docs/topics/mta-web-seeding-instances.adoc

```diff
@@ -0,0 +1,135 @@
+// Module included in the following assemblies:
+//
+// * docs/rules-development-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-seeding-instances_{context}"]
+= Seeding instances
```

Comment:
@CarmiWisemon 
I would suggest 'Seeding an instance'.

Reply:
Thank you!